crt_portal/cts_forms/templates/forms/errors.html

```diff
@@ -23,17 +23,46 @@ <h1 class="padding-top-2 padding-bottom-1 margin-0 title">
 {% block content %}
 <section class="grid-container">
   <div class="grid-row">
-  <div class="tablet:grid-col-8 tablet:grid-offset-2">
-    <div class="crt-success-card">
-      <div class="crt-success-card__content">
-          <h2>{{ status }}<img src="{% static 'img/alerts/error-red.svg' %}" alt="Error indicator with exclamation mark in red circle" width="50px" height="50px" class="padding-top-3"></h2>
-          <h3>{{ message }}</h3>
-          <p>{{ helptext }}</p>
+    <div class="tablet:grid-col-8 tablet:grid-offset-2">
+      <div class="crt-error-card">
+        <div class="crt-error-card__content">
+          <div class="display-flex flex-column padding-bottom-3">
+            <h1 class=".h1__display">{{ status }}</h1>
+            <span>{{ message }}</span>
+            <span class="help-text__small">{{ helptext }}</span>
+          </div>
+          <h2 class="error-subheading">If you need to contact us immediately, you can reach us by:</h2>
+          <div class="crt-error-box">
+            <img src="{% static "img/at-sign.svg" %}" />
+            <span>
+              <li>Email: <a href="mailto:Ask.CRT@crt.usdoj.gov">ask.CRT@crt.usdoj.gov</a></li>
+            </span>
+          </div>
+          <div class="crt-error-box">
+            <img src="{% static "img/phone_70.svg" %}" />
+            <span class="display-flex flex-column">
+              <li>(202) 514-0716</li>
+              <li>Telephone Device for the Deaf</li>
+              <li>(TTY) (202) 514-0716</li>
+            </span>
           </div>
+          <div class="crt-error-box">
+            <img src="{% static "img/mail_70.svg" %}" />
+            <span class="display-flex flex-column">
+              <li>U.S. Department of Justice</li>
+              <li>Civil Rights Division</li>
+              <li>950 Pennsylvannia Avenue, NW</li>
+              <li>Washington, D.C. 20530-0001</li>
+            </span>
+          </div>
+          <h2 class="error-subheading">Other information</h2>
+          <p><a href="https://civilrights.justice.gov/report">File a complaint</a><p>
+          <p><a href="https://civilrights.justice.gov/#about-the-division">About the Civil Rights Division</a><p>
+          <p>If you or someone else is in immediate danger, please call 911 or your local police.</p>
```

Comment:
Might want to add a tel:911 link here? We also have this in the landing page, so we can copy the translation verbatim as well:

```
{% trans 'If you or someone else is in immediate danger, <a href="tel:911">please call 911 or local police.</a>' %}
```